src/external/token/interfaces/IERC20Issuance_blacklist_v1.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+// Imports
+
+// Internal
+import {IERC20Issuance_v1} from "@ex/token/IERC20Issuance_v1.sol";
+
+/**
+ * @title   ERC20 Issuance Token with Blacklist Interface
+ *
+ * @notice  Interface for an ERC20 token that extends standard issuance functionality
+ *          with blacklisting capabilities, allowing for address-based access control
+ *          to token operations.
+ *
+ * @dev     This interface inherits from:
+ *              - IERC20Issuance_v1
+ *          Key features:
+ *              - Individual address blacklisting
+ *              - Batch blacklist operations
+ *              - Blacklist status queries
+ *          All blacklist operations should be restricted to authorized roles
+ *          (e.g., owner or admin) in the implementation.
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer to
+ *                          our Security Policy at security.inverter.network or
+ *                          email us directly!
+ *
+ * @custom:version   1.0.0
+ *
+ * @custom:standard-version  1.0.0
+ *
+ * @author  Zealynx Security
+ */
+interface IERC20Issuance_Blacklist_v1 is IERC20Issuance_v1 {
+    //--------------------------------------------------------------------------
+    // Events
+
+    /// @notice Emitted when an address is added to the blacklist
+    /// @param account_ The address that was blacklisted
+    event AddedToBlacklist(address indexed account_);
+
+    /// @notice Emitted when an address is removed from the blacklist
+    /// @param account_ The address that was removed from blacklist
+    event RemovedFromBlacklist(address indexed account_);
+
+    /// @notice Emitted when a blacklist manager role is granted or revoked
+    /// @param account_ The address that was granted or revoked the role
+    /// @param allowed_ Whether the role was granted (true) or revoked (false)
```

Comment:
Small indentation issue (we always press tab after the keyword).
```suggestion
    /// @param  account_ The address that was granted or revoked the role
    /// @param  allowed_ Whether the role was granted (true) or revoked (false)
```

Reply:
done